docs/library/uasyncio.rst

```diff
@@ -68,7 +68,9 @@ Additional functions
     created from it.
 
     If a timeout occurs, it cancels the task and raises ``asyncio.TimeoutError``:
-    this should be trapped by the caller.
+    this should be trapped by the caller.  The task receives
+    ``asyncio.CancelledError`` which may be ignored.  Cleanup code may be run by
```

Comment:
Not 100% obvious where the cleanup code is meant to be... Maybe rephrase slightly to "... which may be ignored or trapped using `try...except`/`try...finally` to run cleanup code".